spec/helpers/visit_helper_spec.rb

```diff
@@ -84,7 +84,9 @@ def visit
     end
 
     it "provides a formatted date for when a response may be sent out" do
-      expect(helper.when_to_expect_reply(Date.parse("2014-10-03"))).to eq("Monday  6 October")
+      Timecop.travel(Date.parse("2014-10-03")) do
+        expect(helper.when_to_expect_reply).to eq("Friday  3 October")
```

Comment:
That's because this is directly testing the output of the helper and not touching the view. @dan-palmer anything we can do to get around the issue @matharden's describing and get rid of the stupid extra space? (If it's a hassle, don't bother!)